LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 DroidKaigi and contributors.
```

Comment:
Thanks for the suggestions!
We discussed a little.  The "DroidKaigi committee" seems to be better to protect contributors from risk.

Reply:
Thank you for comment.
I'll fix it.